handler/bot.go

```diff
@@ -33,24 +36,24 @@ func NewBot(config *config.Config) (*Bot, error) {
 		githubService:  githubService,
 		channelID:      config.DiscordChannelID,
 		messageID:      config.DiscordMessageID,
+		githubOrg:      config.GithubOrg,
 	}, nil
 }
 
 func (b *Bot) Process() error {
-	// Get open pull request urls
-	pullRequestUrls, err := b.githubService.GetOpenPullRequestUrls()
+	// Get open pull requests
+	pullRequests, err := b.githubService.GetOpenPullRequests(b.githubOrg)
 	if err != nil {
-		return fmt.Errorf("failed to get open pull request urls: %+v", err)
+		return fmt.Errorf("failed to get open pull requests: %+v", err)
 	}
+	// TODO: support listing all open pull requests
+	pullRequests = pullRequests[int(math.Max(0, float64(len(pullRequests)-10))):]
```

Comment:
Since 1.21, Go support builtin [`max`](https://tip.golang.org/ref/spec#Min_and_max) function:
```suggestion
	pullRequests = pullRequests[max(0, len(pullRequests)-10):]
```

Reply:
updated